src/client/components/terminal/index.ts

```diff
@@ -305,6 +305,7 @@ export class TerminalView extends LitElement {
   protected windowSize: IWindowSize
 
   protected rows: number = 10
+  protected errorCode?: number | void
```

Comment:
This doesn't work?
```suggestion
  protected errorCode?: number
```

Reply:
@christian-bromann nop I tried it but it was not working where we get the errorCode returns that type I can for the type if you want. 
![image](https://github.com/stateful/vscode-runme/assets/2709271/773e693f-0b8b-43da-90d1-998894d8db3d)

Reply:
ok, I think it is fine then to keep as is.